FrejaEidClient/src/test/java/com/verisec/frejaeid/client/beans/authentication/init/InitiateAuthetnicationRequestBuildersTest.java

```diff
@@ -16,7 +16,7 @@
 public class InitiateAuthetnicationRequestBuildersTest {
 
     private static final String EMAIL = "email";
-    private static final SsnUserInfo SSN_USER_INFO = SsnUserInfo.create(Country.SWEDEN, "123123123");
```

Comment:
I think we should still user create in tests.

Reply:
Fixed.